app/src/main/java/com/jerboa/datatypes/Others.kt

```diff
@@ -187,3 +190,13 @@ fun getLocalizedStringForInboxTab(
         }
     return returnString
 }
+
+/**
+ * A wrapper to store extra community ban info
+ */
+@Serializable
+data class BanFromCommunityData(
```

Comment:
This is necessary to bundle some info for community bans. site bans don't need this cause `PersonView` has all the data you need.